packages/web/src/common/store/pages/signon/sagas.js

```diff
@@ -742,7 +742,6 @@ function* watchConfigureMetaMask() {
 function* watchFollowArtists() {
   while (
     yield all([
-      take(signOnActions.SIGN_UP_SUCCEEDED),
```

Comment:
See: https://github.com/AudiusProject/audius-protocol/blob/main/packages/web/src/common/store/pages/signon/sagas.js#L499-L501

We don't need to watch `sign_up_succeeded` here. watching `fetchAccountSucceeded` ensures that the account is loaded before we attempt to follow the artists